src/components/devsupport/components/measure/measure.component.tsx

```diff
@@ -61,7 +64,8 @@ export const MeasureElement = (props: MeasureElementProps): MeasuringElement =>
   };
 
   const onUIManagerMeasure = (x: number, y: number, w: number, h: number): void => {
-    const frame: Frame = bindToWindow(new Frame(x, y, w, h), Frame.window());
+    const originY = topInsetEnabled && StatusBar.currentHeight ? y + StatusBar.currentHeight : y;
```

Comment:
Let's make it more readable. 
`const originY = topInsetEnabled ? y + StatusBar.currentHeight || 0 : y`